rgislackbot/app.py

```diff
@@ -62,6 +64,8 @@ def handle_command(slack_client, command, channel):
     # This is where you start to implement more commands!
     if command.startswith("HI"):
         response = "Sure...write some more code then I can do that!"
+    elif command.startswith("repo"):
```

Comment:
might need to rebase some of this since my dispatcher change extracts this logic out of app and into dispatcher.